packages/components/src/higher-order/with-constrained-tabbing/index.js

```diff
@@ -33,6 +33,13 @@ const withConstrainedTabbing = createHigherOrderComponent(
 			} else if ( ! event.shiftKey && event.target === lastTabbable ) {
 				event.preventDefault();
 				firstTabbable.focus();
+			/*
+			 * When pressing Tab and none of the tabbables has focus, the keydown
+			 * event happens on the wrapper div: move focus on the first tabbable.
+			 */
+			} else if ( ! tabbables.includes( event.target ) ) {
```

Comment:
Would it be enough to test `event.target === this.focusContainRef.current` ? More optimized, more explicit to what we are testing.

Or are there other cases where the target of the tab press would itself not be tabbable? I guess might account for elements with tabIndex=-1 within the content? If so, should be update the comment to reflect this?

Reply:
Yep it would be nicer and simpler. Unfortunately, it doesn't work on IE11 where the `event.target` seems to be the clicked children element in the content. See below, in order:
- event.target
- event.currentTarget
- this.focusContainRef.current

![screenshot 94](https://user-images.githubusercontent.com/1682452/46153044-a54ac080-c272-11e8-8bdc-b1ae838e73b5.png)

Reply:
Ah! This is fine then.